test/client/index_test.rb

```diff
@@ -270,15 +222,11 @@
     before do
       suggest = {
         :type            => "completion",
-        :index_analyzer  => "simple",
+        :analyzer        => "simple",
```

Comment:
since you did the other hashes, should you do this one too?

Reply:
Eventually ... I realized I was trying to do "all the things" in a single pull request. I did this one here because of the change lower down where I removed the `es_version_2_x?` check.